nginx-conf/nginx.conf

```diff
@@ -158,13 +158,22 @@ server {
         fastcgi_pass localhost:9000;
     }
 
+    # Scripts/styles/images
     location ~* \.(js|css|png|jpg|jpeg|gif|ico|svg)$ {
         add_header X-Robots-Tag "${X_ROBOTS_TAG}" always;
         try_files $uri @rewrite;
         expires max;
         log_not_found off;
     }
 
+    # Documents
+    location ~* \.(pdf|(xls|doc|pp[st])[xm]?|txt|rtf|csv|key|7z|rar|gz|zip)$ {
+        add_header X-Robots-Tag "${X_ROBOTS_TAG}" always;
```

Comment:
When we launch the site, the `X_ROBOTS_TAG` environment variable will be changed in the production environment, so that other content is indexed.

So I don't think this will have the desired effect - we don't want documents to be indexed in any circumstance, it's not environment dependent.

Reply:
Good spot @keelanfh !